spec/actors/curation_concerns/generic_file_actor_spec.rb

```diff
@@ -9,14 +9,28 @@
   let(:uploaded_file) { fixture_file_upload('/world.png', 'image/png') }
 
   describe 'creating metadata and content' do
-    let(:batch_id) { nil }
+    let(:upload_set_id) { ActiveFedora::Noid::Service.new.mint }
```

Comment:
When it was set to nil, it didn't actually test what happens when upload_set_id is set.  Until this commit, you would have actually gotten an ugly fedora error if you set batch_id/upload_set_id to anything. I fixed it and added test coverage for all cases.